R/gitbook.R

```diff
@@ -284,3 +284,16 @@ download_filenames = function(config) {
   }
   if (length(downloads)) I(downloads)
 }
+
+check_gb_config = function(config) {
+  # to ensure backward compatibility with 0.14 and earlier
+  if (isTRUE(config[["sharing"]][["google"]]) ||
```

Comment:
This check breaks "`sharing: no`": https://travis-ci.org/hadley/mastering-shiny/builds/611371764#L721

Reply:
yes, I am really sorry about that. 
I worked on a fix as soon as I could yesterday evening and pushed a fix this morning with a simpler approach that should have less impacts. https://github.com/rstudio/bookdown/pull/813

Reply:
Should be fixed by `remotes::install_github('rstudio/bookdown')` now. Thanks!

Reply:
When can we expect these fixes on CRAN? As CRAN version of bookdown breaks the book building process if either `sharing: no` or `sharing: yes` are present.

Reply:
@GegznaV Unfortunately the fix came right after I released a new version of bookdown to CRAN, so I cannot update it again too soon. I wonder how difficult it is for you to install the development version.

Reply:
I was asking as I'm teaching a class of students. A part them downloaded the broken version of **bookdown**. I have to give recommendations on how to solve the issue and I usually prefer CRAN versions of packages. But as I see, in this situation, the GitHub version is the only option.

Reply:
I see. The earliest time I can make the next CRAN release is Nov 20 (I'll do it then). Thanks for your patience!